lib/constructs/attribute.js

```diff
@@ -51,8 +51,8 @@ class Attribute {
         throw new Error("Unknown reflector type: " + this.idl.idlType.idlType);
       }
       const attrName = shouldReflect.rhs && shouldReflect.rhs.value.replace(/_/g, "-") || this.idl.name;
-      getterBody = reflector[this.idl.idlType.idlType].get(objName, attrName);
-      setterBody = reflector[this.idl.idlType.idlType].set(objName, attrName);
+      getterBody = reflector[this.idl.idlType.idlType].get(objName, attrName.toLowerCase());
+      setterBody = reflector[this.idl.idlType.idlType].set(objName, attrName.toLowerCase());
```

Comment:
This really should do only ASCII lowercase, although I guess in our case it'll be fine since no IDL attributes are non-ASCII.

(A refactoring project I've meant to do for a while is publish an ASCII string manipulation package with the various utils we've been re-creating in lots of jsdom projects.)